server/multitenancy/test/tenant_index.test.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+describe('Tenant index template', () => {
```

Comment:
Thanks for adding a test, is there any way to test the broader scenario has been fixed, e.g. since the order is high when a new index is created it gets X value instead of Y value?

Reply:
The template and order are parts of OpenSearch core. security-dashboards-plugin only calls the API to create template. It’s sufficient to make sure that the highest order is being set.